tests/cases/fourslash/codeFixDeleteUnmatchedParameter3.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path='fourslash.ts' />
+
+// @filename: a.ts
+/////**
+//// * @param {number} a
+//// * @param {string} b
+//// * @param {number} c
+//// */
+////function foo(a: number, c: number) {}
+
+verify.codeFix({
+    description: [ts.Diagnostics.Remove_unused_parameter_0.message, "b"],
+    index: 0,

```

Comment:
Does this test work if you omit the index, i.e. to assert that this is the only code fix (such that it isn't offered elsewhere)?

Reply:
In this case, we have several code fixes, some of them are not related to this QF. I can add a test to check the list of available code fixes.

Reply:
Ah, darn. I was just hoping for a straightforward way to test a negative, since these tests don't seem to be able to use ranges to assert which fixes happen where (maybe I'm wrong on that front).